src/utils/mod.rs

```diff
@@ -123,11 +123,21 @@ pub fn load_gbassembly_info(input_csv: String) -> Result<(Vec<GBAssemblyData>, u
     // Check column names
     let header = rdr.headers()?;
     let expected_header = vec!["accession", "name"];
-    if header != expected_header {
-        return Err(anyhow!(
-            "Invalid column names in CSV file. Columns should be: {:?}",
-            expected_header
-        ));
+
+    for h in expected_header.iter() {
+        if !header.iter().any(|e| *h == e) {
+            return Err(anyhow!(
+                "Missing column name '{}' in CSV file. Columns should be: {:?}",
+                h,
+                expected_header
+            ));
+        }
+    }
+
+    for h in header.iter() {
+        if !expected_header.iter().any(|e| h == *e) {
+            eprintln!("WARNING: extra column '{}' in CSV file. Ignoring.", h);
+        }
```

Comment:
I'm worried this accepts input CSVs that contain all columns, but not in the right order. The input code currently uses column indexes, not column names, to read the CSV. This has been on my long list to improve, will make an issue for it now.

For this PR, we could just check for the full expected header first, then allow extra columns? Lmk if you want me to take this on.